packages/gestalt/src/Pog.js

```diff
@@ -89,14 +94,8 @@ export default function Pog(props: Props): React.Node {
 
   return (
     <div className={classes} style={inlineStyle}>
-      {/*
-        We're explicitly setting an empty string as a label on the Icon since we
-        already have an aria-label on the button container.
-        This is similar to having empty `alt` attributes:
-        https://davidwalsh.name/accessibility-tip-empty-alt-attributes
-      */}
       <Icon
-        accessibilityLabel=""
+        accessibilityLabel={accessibilityLabel || ''}
```

Comment:
@jackhsu978 since we already set `accessibilityLabel = '',` on line 61, we no longer need the fallback here.